src/style.css

```diff
@@ -1,3 +1,33 @@
+/* stylelint-disable font-family-name-quotes */
 body {
+  background-color: #858585;
+  font-family: 'Roboto', sans-serif;
   margin: 0;
 }
+
+.nav {
+  justify-content: flex-end;
+  font-style: 15px;
+  background-color: #ffff;
+  display: flex;
+  align-items: center;
+}
+
+.nav__link {
+  padding: 25px;
+  text-decoration: none;
+}
+
+.aimg {
+  position: absolute;
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.is_active::after {
+  content: '';
+  height: 4px;
+  background-color: black;
+  display: block;
+  color: #00acdc;
```

Comment:
The `color` property in the `::after` pseudo-element has no effect because it's used for text, and here the content is an empty string. Remove it if you're not going to add text content.